plugin-server/src/main/ingestion-queues/on-event-handler-consumer.ts

```diff
@@ -40,11 +39,36 @@ export const startOnEventHandlerConsumer = async ({
 }
 
 export const buildOnEventIngestionConsumer = ({ hub, piscina }: { hub: Hub; piscina: Piscina }) => {
-    return new IngestionConsumer(
+    return new KafkaJSIngestionConsumer(
         hub,
         piscina,
         KAFKA_EVENTS_JSON,
         `${KAFKA_PREFIX}clickhouse-plugin-server-async`,
         eachBatchAsyncHandlers
     )
 }
+
+export function makeHealthCheck(queue: KafkaJSIngestionConsumer) {
```

Comment:
Moved over from `analytics-events-ingestion-consumer.ts`, while waiting to be deleted once we move this one to rdkafka too